### ISSUE TITLE
Issue to pay Slack reported bug that got fixed internally before new GH was created. 

### ISSUE THREAD

**Issue by mallenexpensify** [Expensify/App#9363]
From this Slack thread https://expensify.slack.com/archives/C01GTK53T8Q/p1653886712901589

@sobitneupane reported a bug that Applause reported after that ended up getting fixed internally so Sobit is owed $250 for reporting.

**Comment by mallenexpensify** [comment]
@sobitneupane hired you, can you accept then I'll pay $250 for reporting the issue? https://www.upwork.com/jobs/~01a76a5fbc7fcb7a60

Sorry @strepanier03 , I should have assigned myself before you were auto-assigned triage, no action needed from you

**Comment by sobitneupane** [comment]
> @sobitneupane hired you, can you accept then I'll pay $250 for reporting the issue? https://www.upwork.com/jobs/~01a76a5fbc7fcb7a60

Accepted.

**Comment by mallenexpensify** [comment]
Paid @sobitneupane $250. Leaving this open til the below is resolved. 

hmm.... good question @Puneet-here , thanks for asking.  

To make sure I understand correctly.  From @sobitneupane 's bug report [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1653886712901589) the reported version is v1.1.68-2, are you saying that, on May 29th, when this was reported, that v1.1.68-2 wasn't a production release?  Based on the issue QA reported ([link](https://github.com/Expensify/App/issues/9223)), that's what it looks like
<img width="275" alt="image" src="https://user-images.githubusercontent.com/22508304/172961019-d4da83c9-8475-4575-bacf-4146837eff19.png">

So.. if it's on staging, then we would compensate based on[ CONTRIBUTING.md here](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#:~:text=If%20the%20issue%20is%20just%20in%20dev%20then%20it)

The one piece I'm unsure on is the regression part, if the above doesn't answer you question can you rephrase and/or expand on the regression part?

**Comment by Puneet-here** [comment]
>So.. if it's on staging, then we would compensate based on[ CONTRIBUTING.md here](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#:~:text=If%20the%20issue%20is%20just%20in%20dev%20then%20it)

Thank for answering.

>The one piece I'm unsure on is the regression part

Oh I was just asking if we also paying contributors for reporting regression before it hits the staging because this one was reported before getting to staging but I just found after few hours of reporting it was deployed to staging. 

>if the above doesn't answer you question can you rephrase and/or expand on the regression part?

it's good to be closed now. Thank you.

**Comment by mallenexpensify** [comment]
Thanks again for raising @Puneet-here , I brought up internally for 👀, if there's an update we'll address in #expensify-open-source.
